apps/test-watchdog/src/main.cpp

```diff
@@ -0,0 +1,13 @@
+
+#include "WatchdogWrapper.h"
+#include "mbed.h"
+
+std::chrono::milliseconds countdown_ms = 1000ms;
+std::chrono::milliseconds pet_ms       = 200ms;
+
+int main() {
+  Utility::WatchdogWrapper::logResetReason();
+  Utility::WatchdogWrapper::startWatchdog(countdown_ms, pet_ms);
+
```

Comment:
Have the program sleep for a bit before doing MBED_ASSERT(false), to test that the watchdog petting works